gateway-service/src/chef/chef.service.ts

```
@@ -54,21 +53,32 @@ export class ChefService {
   }
 
-  async updateProfileChef(body: ChefModel, id: number): Promise<any> {
+  async updateProfileChef(payload: any, chefId: number): Promise<any> {
     try {
       const result = await lastValueFrom(
-        this.chefClient.send('chef/updateProfile', { body, id }),
+        this.chefClient.send('chef/updateProfile', { payload, chefId }),
       );
       return result;
     } catch (error) {
       throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)